libs/sr/formula-ui/src/char/tagFieldMap.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  compileTagMapValues,
+  TagMapSubset,
+} from '@genshin-optimizer/pando/engine'
+import type { TagField } from '@genshin-optimizer/pando/ui-sheet'
+import { isTagField } from '@genshin-optimizer/pando/ui-sheet'
+import type { Tag } from '@genshin-optimizer/sr/formula'
+import { keys } from '@genshin-optimizer/sr/formula'
+import { uiSheets } from './sheets'
+
+const tagValue: Array<{ tag: Tag; value: TagField }> = []
+Object.values(uiSheets).forEach((sheet) => {
+  Object.values(sheet).forEach(({ documents }) => {
+    documents.forEach((document) => {
+      if (document.type === 'fields')
+        document.fields.forEach((field) => {
+          if (!isTagField(field)) return
+          tagValue.push({ tag: field.fieldRef, value: field })
+        })
+    })
+  })
+})
+
+//TODO: This breaks? wtf?
```

Comment:
![image](https://github.com/user-attachments/assets/0a43bdc1-c4f8-4631-9867-0605f9a01460)
no idea what this error means